packages/sdk/src/rest/CicsCmciRestError.ts

```diff
@@ -19,12 +21,16 @@ export class CicsCmciRestError extends ImperativeError {
   RESPONSE_2: number;
   RESPONSE_1_ALT: string;
   RESPONSE_2_ALT: string;
+  FEEDBACKRESP: number;
+  FEEDBACKRESP_2: number;
+  FEEDBACK_ACTION: string;
+  EIBFN_ALT: string;
+  FEEDBACKRESP_ALT: string;
 
-  constructor(msg: string, resultSummary: ICMCIResponseResultSummary) {
-    super({
-      msg,
-    });
+  constructor(msg: string , resultSummary: ICMCIResponseResultSummary, errorFeedback?: ICMCIResponseErrors) {
```

Comment:
Wondering if you can just take 1 `ICMCIAPIResponse` parameter rather than `ICMCIResponseResultSummary` and `ICMCIResponseErrors` as they share a parent right? I see you're doing this elsewhere...

```
throw new CicsCmciRestError(CicsCmciMessages.cmciRequestFailed.message, apiResponse.response.resultsummary, apiResponse.response.errors);
```

Is there a reason you can't do this?
```
throw new CicsCmciRestError(CicsCmciMessages.cmciRequestFailed.message, apiResponse.response);
```